### PR TITLE
Gatsby upgrade

### PR DESCRIPTION
**Pull request by winkerVSbecks**

### CODE REVIEW

File: src/components/basics/Pre.tsx [hunk 1]
```diff
@@ -43,6 +43,7 @@ export function Pre({ children }) {
         />
       );
     }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```
Comment (kylegach) [inline src/components/basics/Pre.tsx:46]: The amount of times I'm seeing this rule disabled is a bit worrying. Should we have a follow-up to investigate further?

**Review by kylegach — APPROVED** [review]
Good work! One minor comment; no blockers. 🚢 it!

File: src/components/basics/Pre.tsx (hunk 1 above)
Comment (winkerVSbecks) [reply src/components/basics/Pre.tsx:46]: Yea I agree. I do want to fix that but maybe as the next pass. The code didn't change I just updated eslint which started flagging for this.

**Comment by winkerVSbecks** [comment]
@kylegach Going to merge for now and will revisit the exhaustive deps warning

**Pull request merged**